tutorials/README.md

```diff
@@ -0,0 +1,10 @@
+## Tutorials
+This folder contains tutorial notebooks for DIANNA.
+A tutorial is avaiable for several combinations of data modality and explanability method.
+Click one of the links in the table to directly go to a notebook:
+
+|            | images                     | text |
+| :--        | :--:                       | :--:  |
+| LIME       | [link](lime_images.ipynb)  | []() |
+| RISE       | []()                       | []() |
+| KernelShap | []()                       | []() |
```

Comment:
Elena and I just worked on the Readme. There we have the same table, but transposed. We thought it's more common to have the XAI methods in the columns (although both ways make perfect sense). For consistency, could you do the same for this one?

Reply:
Done!